.github/workflows/ci.yml

```diff
@@ -194,4 +194,4 @@ jobs:
 
     - name: End-to-end tests
       run: |
-        REGISTRY="quay.io/coreos" make e2e
+        REGISTRY="k8s.gcr.io/kube-state-metrics" make e2e
```

Comment:
Does this one still have an effect at all or can we simply remove it?

Reply:
```suggestion
        make e2e
```

Reply:
Yeah, I don't think it has an effect